stl/inc/xmemory

```diff
@@ -1187,6 +1231,26 @@ inline void _Container_base12::_Swap_proxy_and_iterators(_Container_base12& _Rig
     }
 }
 
+inline void _Container_base12::_Swap_proxy_and_iterators_locked(_Container_base12& _Right) noexcept {
+    _Lockit _Lock(_LOCK_DEBUG);
+    _Swap_proxy_and_iterators_unlocked(_Right);
+}
+
+_CONSTEXPR20_DYNALLOC void _Container_base12::_Swap_proxy_and_iterators(_Container_base12& _Right) noexcept {
+#if _ITERATOR_DEBUG_LEVEL != 2
+    _Swap_proxy_and_iterators_unlocked(_Right);
+#else // ^^^ _ITERATOR_DEBUG_LEVEL != 2 ^^^ / vvv _ITERATOR_DEBUG_LEVEL == 2 vvv

```

Comment:
We conventionally prefer to avoid negated tests unless there's a code clarity reason. (This avoids mental "double negatives".) Here, changing the test to be `IDL == 2` and swapping the blocks would avoid the negation in `!=`.